docker/local-metadata/docker-compose-postgres.yml

```diff
@@ -124,6 +124,11 @@ services:
       # Airflow SSL Configurations
       AIRFLOW_VERIFY_SSL: ${AIRFLOW_VERIFY_SSL:-"no-ssl"}
       AIRFLOW_SSL_CERT_PATH: ${AIRFLOW_SSL_CERT_PATH:-""}
+      # ElasticSearch Configurations
```

Comment:
Can you please also group `ELASTICSEARCH_HOST` env that is placed at Line 64 here ?